PointOfDispensingApp/src/components/PatientLookupPage.vue

```diff
@@ -8,20 +8,33 @@
         <v-divider></v-divider>
       </v-col>
     </v-row>
-    <v-row>
-      <v-col cols="12">
-        <div class="font-weight-medium secondary--text">
-          Retrieve the patient's record by scanning their QR code.
-        </div>
-      </v-col>
-    </v-row>
-    <v-row>
-      <v-col cols="12">
-        <v-btn color="accent" @click="scanQrCode">
-          Scan QR code
-        </v-btn>
-      </v-col>
+    <v-row> 
+        <v-col cols="12">
+          <div class="font-weight-medium secondary--text">Retrieve the patient's record by scanning their QR code.</div>
+        </v-col>
     </v-row>
+    
+  <template>
+  <v-row><div>
+    <p class="error" v-if="isCameraOn && noFrontCamera">
+      You don't seem to have a front camera on your device
+    </p>
+
+    <p class="error" v-if="isCameraOn && noRearCamera">
+      You don't seem to have a rear camera on your device
+    </p></div></v-row>
+  <v-row>
+    <v-col cols="6">
+    <v-btn @click="scanQrCode()" color="accent"> Scan QR Code
+    </v-btn></v-col>
+    <v-col cols="6">
+    <div v-if="isCameraOn && !noFrontCamera && !noRearCamera"><v-btn @click="switchCamera" color="accent"> Switch Camera
+    </v-btn></div></v-col></v-row>
```

Comment:
I'd recommend spacing out the tags more so that you can more clearly see what contains/encloses what.
This makes it easier to read and get a sense of the structure.
Ex. 
`<v-row>
    <v-col cols="6">
      <v-btn @click="scanQrCode()" color="accent"> Scan QR Code </v-btn>
    </v-col>
    <v-col cols="6">
      <div v-if="isCameraOn && !noFrontCamera && !noRearCamera">
        <v-btn @click="switchCamera" color="accent"> Switch Camera </v-btn>
      </div>
    </v-col>
  </v-row>`